plugins/bigquery/dbt/include/bigquery/macros/materializations/incremental.sql

```diff
@@ -112,7 +112,7 @@
       {% endif %}
       {% set build_sql = create_table_as(False, target_relation, sql) %}
   {% else %}
-     {% set dest_columns = adapter.get_columns_in_relation(existing_relation) %}
+     {% set dest_columns = config.get('update_columns') %}
```

Comment:
From your comment on #1862:
> We'd probably want to default to adapter.get_columns_in_relation if the update_columns config parameter isn't specified, right?
Yes, agreed. So I think you'll want this to look like:
```suggestion
     {% set dest_columns = config.get('update_columns', none) %}
     {% if dest_columns is none %}
         {% set dest_columns = adapter.get_columns_in_relation(existing_relation) %}
     {% endif %}
```